src/reducers/local-storage.js

```diff
@@ -0,0 +1,21 @@
+export const getReduxState = () => {
```

Comment:
This is named `getReduxState` but isn't getting state from redux

Reply:
I think this can still be renamed to something more appropriate to what it's doing. I'd also recommend moving this into the `utils` folder since it isn't doing anything regarding the reducer.